samples/StaticHtmlComposites/index.html

```diff
@@ -4,26 +4,31 @@
   <title>Basic example for CallComposite and ChatComposite</title>
 </head>
 <body>
-  <div id="video-call"></div>
-  <div id="chat"></div>
-  <script src="./dist/bundle.js"></script>
+  <!-- height need to be set for composite to fit the layout -->
+  <div id="video-call" style="height: 500px;"></div>
+  <div id="chat" style="height: 500px;"></div>
+  <script src="./dist/chatComposite.js"></script>
+  <script src="./dist/callComposite.js"></script>
+  <script src="./dist/service.js"></script>
   <script type="module">
-    const { user, token } = await app.createUserWithToken();
+    const { user, token } = await service.createUserWithToken();
+    // Call composite sample code starts here
     const displayName = 'Bob';
 
-    const callAdapter = await app.loadCallComposite({
+    const callAdapter = await callComposite.loadCallComposite({
       containerId: 'video-call',
       groupId: '6e606975-7004-4c03-be37-e0851ed1850b', // any GUID
```

Comment:
do we think we might want to bring in a dependency which randomly generates a guid? It could be a privacy concern if someone uses this and then sees other random people in the call

Reply:
Good catch! Haha, I can imagine the scenes and it feels scary